dirs/dirs_test.go

```diff
@@ -47,3 +48,29 @@ func (s *DirsTestSuite) TestStripRootDir(c *C) {
 	// strip only works on paths that begin with the global root directory
 	c.Check(func() { dirs.StripRootDir("/other/foo/bar") }, Panics, `supplied path is not related to global root "/other/foo/bar"`)
 }
+
+func (s *DirsTestSuite) TestClassicConfinementSupport(c *C) {
+	dirs.SetRootDir("/")
+	c.Assert(dirs.SupportsClassicConfinement(), Equals, true)
+	dirs.SetRootDir("/alt")
+	c.Assert(dirs.SupportsClassicConfinement(), Equals, false)
+}
+
+func (s *DirsTestSuite) TestClassicConfinementSupportOnSpecificDistributions(c *C) {
+	for _, current := range []struct {
```

Comment:
The essence yes, but not the switch() statement which checks for different distributions. I feel better testing that explicitly so we see when this gets broken for a particular distribution.